Sources/KlaviyoSwift/KlaviyoModels.swift

```diff
@@ -70,7 +70,16 @@ public struct Event: Equatable {
                 value: Double? = nil,
                 time: Date? = nil,
                 uniqueId: String? = nil) {
-        _profile = AnyCodable(profile ?? [:])
+        var profile = profile
```

Comment:
This is probably temporary. We move some old parameters from the profile dictionary to the identifers structure so it gets picked up in the api request later. If any of these are absent they were be overwritten by the state value or they will overwrite state.